Index: app/src/main/java/com/example/meetyou/ChoosePartnerParametersActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/meetyou/ChoosePartnerParametersActivity.java b/app/src/main/java/com/example/meetyou/ChoosePartnersParametersActivity.java
rename from app/src/main/java/com/example/meetyou/ChoosePartnerParametersActivity.java
rename to app/src/main/java/com/example/meetyou/ChoosePartnersParametersActivity.java
--- a/app/src/main/java/com/example/meetyou/ChoosePartnerParametersActivity.java	
+++ b/app/src/main/java/com/example/meetyou/ChoosePartnersParametersActivity.java	
@@ -15,7 +15,7 @@
 import com.example.meetyou.MYFiles.Users;
 import com.example.meetyou.databinding.ActivityChoosePartnersParametersBinding;
 
-public class ChoosePartnerParametersActivity extends AppCompatActivity {
+public class ChoosePartnersParametersActivity extends AppCompatActivity {
 
     ActivityChoosePartnersParametersBinding binding;
     DatabaseHelper databaseHelper;
@@ -65,7 +65,7 @@
                 Users.updateUserFindHeight(getUID(), selectedHeight);
                 Users.updateUserFindWeight(getUID(), selectedWeight);
 
-                Intent intent = new Intent(ChoosePartnerParametersActivity.this, StartActivity.class);
+                Intent intent = new Intent(ChoosePartnersParametersActivity.this, StartActivity.class);
                 intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
                 startActivity(intent);
                 finish();
